app-misc/bgrep/Manifest

```diff
@@ -1 +1,2 @@
+DIST bgrep-0.2_p20210321.tar.gz 4854 BLAKE2B 85c9171572e440f1e5def3a902a4adb1a3339706847013c6ff139745a819d9eab48a9449bf89b631cfcfe0081365240eefcae46a68064f710178db6d2db50037 SHA512 a9488ee9fd68a1b0e9924cbdf1e36fb3517cdceee4b57a27ef148ccd2c3f51942f8a57b41d47a2cb5b65b376b0b87c92ca4ee5d7053714a725c57ffbabbf12b6
```

Comment:
forgot to say, that we version the snapshot by the date of upstream commit, not by the date when ebuild was written. For obvious reason, helps to identify the commit when the snapshot was made, and prevents useless "version" bumps by writing the same ebuild on different date.

I fixed it while merging, but as a learning example for you.

Reply:
Thanks :)